### PR TITLE
Use mjs for index

### PR DESCRIPTION
**Pull request by hobbitronics**
- change: entry point to index.mjs
- remove: icons and update README on how to import them
- fix: documents reference causing error for svelte-kit SSR

image
got theme working by installing material and sass
image

### CODE REVIEW

**Comment by hobbitronics** [comment]
Theming and typography and material icons are still not working properly.

File: components/mdc/icons/index.js [hunk 1]
```diff
@@ -1,6 +1,9 @@
 import './_index.css'
+import { browser } from '$app/env'
```
Comment (longrunningprocess) [inline components/mdc/icons/index.js:2]: isn't this going to create a dep on kit?

File: index.mjs [hunk 2]
```diff
@@ -22,12 +22,12 @@ import {
   Tooltip,
   TopAppBar,
   setNotice,
-} from './components/mdc'
+} from './components/mdc/index.js'
```
Comment (longrunningprocess) [inline index.mjs:25]: I'm surprised these are still needed?

Comment (hobbitronics) [reply index.mjs:25]: yeah, it didn't like it for some reason. Storybook was the complainer.

**Comment by hobbitronics** [comment]
closing this as it is no longer a major version change. Will reopen another PR with same changes but correct commit message.

File: components/mdc/icons/index.js (hunk 1 above)
Comment (hobbitronics) [reply components/mdc/icons/index.js:2]: I think it's already included. I might need to test it in non kit stuff though.

**Pull request closed without merging**